src/models/transactions/offerCancel.ts

```diff
@@ -0,0 +1,21 @@
+import { ValidationError } from "../../common/errors";
+import { BaseTransaction, verifyBaseTransaction } from "./common";
+
+export interface OfferCancel extends BaseTransaction {
+    TransactionType: "OfferCancel";
+    OfferSequence: number;
+}
+
+/**
+ * Verify the form and type of an OfferCancel at runtime.
+ * 
+ * @param tx - An OfferCancel Transaction
+ * @returns - Void.
```

Comment:
I think it makes more sense to have return type be `boolean` instead of `void`. We can `return true` at the end of this function.